src/uu/ls/src/ls.rs

```diff
@@ -639,41 +572,48 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             Arg::with_name(options::format::ONE_LINE)
                 .short(options::format::ONE_LINE)
                 .help("List one file per line.")
-                .multiple(true)
+                .multiple(true),
         )
         .arg(
             Arg::with_name(options::format::LONG_NO_GROUP)
                 .short(options::format::LONG_NO_GROUP)
-                .help("Long format without group information. Identical to --format=long with --no-group.")
-                .multiple(true)
+                .help(options::format::LONG_NO_GROUP_HELP)
```

Comment:
same here

Reply:
Putting the text here will make `cargo fmt` not work. So I am keeping it here.

Reply:
splitting it on several lines will probably fix it

Reply:
Nice, I have reduced the fix to just formatting changes with the method you suggested.